api/v1/checks.go

```diff
@@ -779,6 +779,23 @@ func (c GitHubCheck) GetEndpoint() string {
 	return strings.ReplaceAll(c.Query, " ", "-")
 }
 
+type GitLabCheck struct {
+	Description    `yaml:",inline" json:",inline"`
+	Templatable    `yaml:",inline" json:",inline"`
+	ConnectionName string       `yaml:"connection,omitempty" json:"connection,omitempty"`
+	Repo           string       `yaml:"repo" json:"repo"`
+	Username       string       `yaml:"username" json:"username"`
+	PasswordSecret types.EnvVar `yaml:"passwordSecret,omitempty" json:"passwordSecret"`
```

Comment:
These names are more consistent with other checks:
```suggestion
	Description    `yaml:",inline" json:",inline"`
	Templatable    `yaml:",inline" json:",inline"`
	ConnectionName string       `yaml:"connection,omitempty" json:"connection,omitempty"`
	Repository           string       `yaml:"repository" json:"repository"`
	Username       types.EnvVar       `yaml:"username" json:"username"`
	Password types.EnvVar `yaml:"password,omitempty" json:"password"`
```

Reply:
You're right, these names are better. I've changed them, thank you.